test/unit/api/index.js

```diff
@@ -145,30 +149,49 @@ describe('api', function() {
     });
   }));
 
-  describe('resolved promise', function() {
+  describe('resolved promise', function () {
 
-    describe('configuring', function() {
-      it('GETs the configured url for system-number', function() {
+    describe('configuring', function () {
+      it('GETs the configured url for system-number', function () {
         requestGet.yields(null, {statusCode: 200}, JSON.stringify(response));
 
         return api.read({
           'system-number': '1234'
-        }).then(function() {
-          request.get.should.have.been.calledWith('http://testhost.com:1111/api/v0/events/birth/1234');
+        }, 'mrs-caseworker').then(function () {
+          request.get.should.have.been.calledWith({
+            url: 'http://testhost.com:1111/api/v0/events/birth/1234',
+            headers: usernameHeader
```

Comment:
Might be better to have this as the string literal, 'mrs-caseworker'.
'What will test the tests?' etc.